jobs/webcompat-kb/webcompat_kb/bugzilla.py

```diff
@@ -244,74 +244,43 @@ def missing_initial_add(self) -> bool:
 
 HistoryByBug = Mapping[BugId, Sequence[BugHistoryEntry]]
 
-BUG_QUERIES: Mapping[str, dict[str, str]] = {
-    "site_reports_wc": {
+BUG_QUERIES: Mapping[str, dict[str, str | list[str]]] = {
+    "webcompat_product": {
+        "component": [
+            "Knowledge Base",
+            "Privacy: Site Reports",
+            "Site Reports",
+            "Interventions",
+        ],
         "product": "Web Compatibility",
-        "component": "Site Reports",
-        "f1": "OP",
-        "f2": "bug_status",
-        "o2": "changedafter",
-        "v2": "2020-01-01",
-        "j1": "OR",
-        "f3": "resolution",
-        "o3": "isempty",
-        "f4": "CP",
+        "j_top": "OR",
+        "f1": "bug_status",
+        "o1": "changedafter",
+        "v1": "2020-01-1",
+        "f2": "resolution",
+        "o2": "isempty",
     },
-    "site_reports_other": {
+    "webcompat_other": {
         "f1": "product",
         "o1": "notequals",
         "v1": "Web Compatibility",
-        "f2": "keywords",
-        "o2": "substring",
-        "v2": "webcompat:site-report",
-    },
-    "knowledge_base": {
-        "product": "Web Compatibility",
-        "component": "Knowledge Base",
-    },
-    "platform_bugs": {
-        "f1": "product",
-        "o1": "notequals",
-        "v1": "Web Compatibility",
-        "f2": "keywords",
-        "o2": "substring",
-        "v2": "webcompat:platform-bug",
-    },
-    "interventions": {
-        "product": "Web Compatibility",
-        "component": "Interventions",
-    },
-    "other": {
-        "f1": "product",
-        "v1": "Web Compatibility",
-        "o1": "notequals",
-        "keywords": "webcompat:",
-        "keywords_type": "regexp",
-    },
-    "parity": {
-        "f1": "OP",
-        "f2": "bug_status",
-        "o2": "changedafter",
-        "v2": "2020-01-01",
-        "j1": "OR",
-        "f3": "resolution",
-        "o3": "isempty",
-        "f4": "CP",
-        "f5": "keywords",
-        "o5": "regexp",
-        "v5": "parity-",
-    },
-    "site_reports_etp": {
-        "product": "Web Compatibility",
-        "component": "Privacy: Site Reports",
-        "f1": "OP",
-        "f2": "bug_status",
-        "o2": "changedafter",
-        "v2": "2020-01-01",
-        "j1": "OR",
-        "f3": "resolution",
-        "o3": "isempty",
-        "f4": "CP",
+        "f2": "OP",
+        "j2": "OR",
+        "f3": "keywords",
+        "o3": "casesubstring",
+        "v3": "webcompat:",
+        "f4": "keywords",
+        "o4": "casesubstring",
+        "v4": "parity-",
+        "f5": "CP",
+        "f6": "OP",
+        "j6": "OR",
+        "f7": "bug_status",
+        "o7": "changedafter",
+        "v7": "2020-01-1",
```

Comment:
This appears to work, but I think it should be `2020-01-01` just so it looks less weird. (Same some lines above)